docs/Administration.rst

```diff
@@ -4639,13 +4642,128 @@ You can review the list of kudos received and sent from the tab "Kudos List" of
 
 |image336|
 
-
 You can filter per period type and select the period in which you want to see all sent (and recieved) kudos.
 
 |image335|
 
 
 
+.. _ManagingRewards:
+
+================
+Managing Rewards
+================
+
+You can manage the rewards sending from "Reward administration" page. 
+To access to this interface, click on "Administration" menu from the top navigation, then "Reward" > "Reward" from the drop-down menu.
+The first thing to do is to setup how rewards will be calculated from the configuration tab. 
+Then, from "Pools" tab you can create pools and assign users to them according to their domains. 
+Since the use of the platform may not be the same from one team to another, and in order to ensure a fair distribution of rewards, you can divide users by pools according to their teams. 
+This will put the members of the same team in competition together but not with all the users of the platform.
+Finally, when everything is setup correctly in both configuration and pools tabs, you can send rewards from the first tab.
+
+
+Configuration
+~~~~~~~~~~~~~
+
+Configuration is the third tab of Reward interface. 
+This tab lets you manage the rewards sending, the periodicity and how rewards will be calculated for users at the end of each period.
+
+|image337|
+
+Choose the periodicity for sending rewards. The possible values are : week, month, quarter, semester or year.
+The interface is divided into two blocks : a block to setup the rewards for kudos and a block to setup the rewards for gamification points.
+For each bloc you can specify a minimal threshold to reward users. The default value is "0", which means that the users are rewarded automatically if they receive kudos or earn any gamification points.
+
+Select the checkbox "Use pools" if you want to use the pools in the rewards calculation and this for kudos and gamification.
```

Comment:
The gamification program lets you optionally leverage pools by checking the "Use Pools" box. It means budget settings are applicable per pool rather than globally.